### PR TITLE
Fix read non-existing file & Readme

### PR DESCRIPTION
**Pull request by thisIsTheFoxe**
Hi.. so sorry, didn't have time to check out #4 earlier..
So, all in all looks pretty good. I like using sets to avoid duplicates, btw 👍

Just 2 things I noticed:
1. Without `--remove-dead` it tries to read the file even if it doesn't exist
2. It might be good to add `--remove-dead` to the Readme instead of my solution it's easier and clearer.

### CODE REVIEW

**Comment by ko973** [comment]
+``` + + +Classification example: +```python: +from transformers import AutoModelForSequenceClassification, AutoTokenizer + +model = AutoModelForSequenceClassification.from_pretrained("ccdv/legal-lsg-base-uncased-4096", + trust_remote_code=True, + pool_with_global=True, # pool with a global token instead of first token +) +tokenizer = AutoTokenizer.from_pretrained("ccdv/legal-lsg-base-uncased-4096") + +SENTENCE = "This is a test for sequence classification. " * 300 +token_ids = tokenizer( + SENTENCE, + return_tensors="pt", + padding="max_length", # Optional but recommended + truncation=True # Optional but recommended + ) +output = model(**token_ids) + +> SequenceClassifierOutput(loss=None, logits=tensor([[-0.3051, -0.1762]], grad_fn=<AddmmBackward>), hidden_states=None, attentions=None) +``` + +## Training global tokens +To train global tokens and the classification head only: +```python: +from transformers import AutoModelForSequenceClassification, AutoTokenizer + +model = AutoModelForSequenceClassification.from_pretrained("ccdv/legal-lsg-base-uncased-4096", + trust_remote_code=True, + pool_with_global=True, #
